taxonomy/tests/test_models.py

```diff
@@ -54,24 +54,24 @@ def test_slug(self):
 
 class TestCuratorVocabulary(LoreTestCase):
     """
-    Tests for the automatically-created curator vocabulary.
+    Tests for automatically-created vocabularies.
     """
     def test_vocab_created(self):
         """
-        A Repository should have the "curation status" vocabulary added
-        automatically upon creation.
+        At the moment there are no vocabularies automatically created
+        though that will change soon.
```

Comment:
Consider mentioning issue #576 as the description of the change.  (Though I suppose this docstring will be short-lived.)